pocs/tests/test_database.py

```diff
@@ -31,14 +34,33 @@ def test_simple_insert(db):
 # Filter out (hide) "UserWarning: Collection not available"
 @pytest.mark.filterwarnings('ignore')
 def test_bad_collection(db):
-    with pytest.raises(AssertionError):
+    with pytest.raises(InvalidCollection):
         db.insert_current('foobar', {'test': 'insert'})
 
-    with pytest.raises(AssertionError):
+    with pytest.raises(InvalidCollection):
         db.insert('foobar', {'test': 'insert'})
 
 
-def test_bad_object(db):
+def test_log_bad_object(db, caplog):
```

Comment:
I know I saw `caplog` fixture somewhere but did it get pulled out of this PR or am I just missing it?